### PR TITLE
Feat/assets/woff transformer

### PR DESCRIPTION
**Pull request by torgeadelin**
# Description
Added
`TTF` -> `WOFF` Font Transform
Changed FileHandler to support multiple encoding configurations.

### CODE REVIEW

File: packages/@momentum-design/icons/config/momentum.json [hunk 1]
```diff
@@ -81,11 +81,30 @@
           "destination": "./dist/fonts",
           "format": {
             "type": "TTF",
+            "encoding": {
+              "read": "utf-8",
+              "write": null
```
Comment (cscheffauer) [inline packages/@momentum-design/icons/config/momentum.json:86]: what is null encoding? doesn't make really sense to me

**Review by cscheffauer — APPROVED** [review]
question on encoding

**Review by chrisadubois — APPROVED** [review]
Small comment. Also produces woff2?

File: packages/@momentum-design/icons/config/momentum.json (hunk 1 above)
Comment (torgeadelin) [reply packages/@momentum-design/icons/config/momentum.json:86]: ttf holds binary information, and that's what we need for woff and woff2. If i don't add the null, it encodes it as utf-8 and when i read it again in the woff transform it's invalid.

File: packages/@momentum-design/builder/src/assets/builder/file-handler.ts [hunk 2]
```diff
@@ -112,9 +118,9 @@ class FileHandler {
    * @param file - File to read data from
    * @returns Promise, which resolves to the file object with data if successful
    */
-  public readFile(file: FileType): Promise<FileType> {
+  public readFile(file: FileType, encoding: string): Promise<FileType> {
```
[the remark this answers is not in the document]
Comment (torgeadelin) [reply packages/@momentum-design/builder/src/assets/builder/file-handler.ts:121]: woff2 is separate

**Pull request merged**